public/javascripts/Admin/src/Admin.GSVLabelView.js

```diff
@@ -521,12 +521,28 @@ function AdminGSVLabelView(admin, source) {
         });
     }
 
+    function _escapeHTML(str) {
```

Comment:
Love that we have this now! Could you add it to `Utilities.js` instead of here? That's imported into most of our pages already, and I think that this could be useful in other places in the future!